viewing_party/party.py

```diff
@@ -1,23 +1,179 @@
 # ------------- WAVE 1 --------------------
 
+from re import I
+
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if title == None or genre == None or rating == None:
+        return None
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+
+    user_data = {
+        "watched": [],
+        "watchlist": [],
+    }
+
+    add_to_watched(user_data, new_movie)

```

Comment:
Not clear why `user_data` dictionary is created and modified with these function calls. I commented them out and ran the tests, which still pass. `user_data` isn't a globally declared variable, so this function is essentially creating a dictionary and modifying it, but not doing anything with the value.